### PR TITLE
"Systematic leniency": allow waiving penalties under certain conditions. Addresses #798.

### PR DESCRIPTION
**Pull request by lgarron**
Ideally, we would like to learn when Delegates believe that it is more fair to waive certain penalties instead of being more strict. This gives them a mechanism to do so. The change requires noting this in the Delegate report so that we can understand which Regulations may need clarifications or revisions in order to ensure fair rulings across the world.

I believe that Delegates are generally in favor of this, although we didn't really get feedback about how to implement it. I believe that this is a sufficiently flexible start that we can refine in the future.

### CODE REVIEW

**Comment by xsrvmy** [comment]
I've not sure if intentionally is the best word here. I would argue that touching camera during inspection is intentional.

File: wca-regulations.md [hunk 1]
```diff
@@ -291,6 +291,10 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 11a3) Equipment malfunction.
 - 11b) If an incident occurs, the WCA Delegate determines an impartial and appropriate course of action.
 - 11d) If the WCA Regulations are not fully clear or if the incident is not covered by the WCA Regulations, then the WCA Delegate must make a decision based on fair sportsmanship (also see [Regulation 11e3](regulations:regulation:11e3)).
+  - 11d1) The WCA Delegate may waive a penalty or award an extra attempt even when Regulation does not directly specify the use of their discretion. This must be noted in the Delegate report, and satisfy the following conditions:
```
Comment (Ivan-Brigidano) [inline wca-regulations.md]:
```suggestion
  - 11d1) The WCA Delegate may waive a penalty or award an extra attempt even when Regulations does not directly specify the use of their discretion. This must be noted in the Delegate report, and satisfy the following conditions:
```

File: wca-regulations.md [hunk 2]
```diff
@@ -291,6 +291,10 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
+    - 11d1a) The competitor did not cause the incident, or did not cause it intentionally.
+    - 11d1b) The incident did not give the competitor a significant advantage for their result.
```
Comment (Ivan-Brigidano) [inline wca-regulations.md]:
```suggestion
    - 11d1b) The incident did not give the competitor an unfair advantage for their result.
```

File: wca-guidelines.md [hunk 3]
```diff
@@ -144,6 +144,7 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-11><incidents><incidents> Article 11: Incidents
 
+- 11d1+) [EXAMPLE] Examples of incidents where it is appropriate to waive a penalty: touching a camera during inspection (see [Regulation 2i2b](regulations:regulation:2i2b)), lightly touching a puzzle at the end of the attempt (see [Regulation A6e](regulations:regulation:A6e)).
```
Comment (Ivan-Brigidano) [inline wca-guidelines.md:148]: Why `lightly` and not `slightly`? Is that because the one you use is American English and the other is British?

**Review by Ivan-Brigidano — CHANGES_REQUESTED** [review]
I would also add:
`11d1a+) [REMINDER] The competitor must not have caused the incident internationally.`
Or something similar.

Note: I didn't find why it fails the CI

File: wca-guidelines.md (hunk 3 above)
Comment (lgarron) [reply wca-guidelines.md:148]: Perhaps. I think "lightly" is definitely more idiomatic.

File: wca-regulations.md (hunk 2 above)
Comment (lgarron) [reply wca-regulations.md]: Good idea, that is more consistent with other with other Regulations.

File: wca-regulations.md (hunk 1 above)
Comment (lgarron) [reply wca-regulations.md]:
```suggestion
  - 11d1) The WCA Delegate may waive a penalty or award an extra attempt even when the relevant Regulation does not directly specify the use of their discretion. This must be noted in the Delegate report, and satisfy the following conditions:
```

**Comment by lgarron** [comment]
> I've not sure if intentionally is the best word here. I would argue that touching camera during inspection is intentional.

You're right, this could be clearer. I've changed this so that it says they must not have tried to gain an advantage.

**Comment by lgarron** [comment]
@Claster, @julesDesjardin: Are you comfortable with this wording?

File: wca-guidelines.md (hunk 3 above)
Comment (lgarron) [reply wca-guidelines.md:148]: "lightly touch" seems to have about 1x as many Google results as "slightly touch", so I think I'd like to stay with that.

**Comment by Ivan-Brigidano** [comment]
I would add the third point

> 11d1c) The competitor must not have caused the incident intentionally.

Touching the camera is not always intentionally, in fact, most cases of competitors touching the camera are because of a knee-jerk reaction. Maybe a guideline could clarify it:

> 11d1c+) [CLARIFICATION] A knee-jerk reaction is not consider an intentionally incident.

Maybe not the best wording thought.

**Comment by lgarron** [comment]
Given that we 

> I would add the third point
> 
> > 11d1c) The competitor must not have caused the incident intentionally.
> 
> Touching the camera is not always intentionally, in fact, most cases of competitors touching the camera are because of a knee-jerk reaction. Maybe a guideline could clarify it:
> 
> > 11d1c+) [CLARIFICATION] A knee-jerk reaction is not consider an intentionally incident.
> 
> Maybe not the best wording thought.

The best I know is "forced" vs. "unforced" error. But that's not a super common phrase.

Given the discussion here and other places, I think I'm okay with leaving this draft out of the 2020 Regulations and refining it in collaboration with Delegates.

**Pull request closed without merging**